tests/custom.rs

```diff
@@ -1,54 +0,0 @@
-// Test that a custom handler works on wasm32-unknown-unknown
-#![cfg(all(
-    target_arch = "wasm32",
-    target_os = "unknown",
-    feature = "custom",
-    not(feature = "js")
-))]
-
-use wasm_bindgen_test::wasm_bindgen_test as test;
-
-use core::num::NonZeroU32;
-use getrandom::{getrandom, register_custom_getrandom, Error};
-
-fn len7_err() -> Error {
-    NonZeroU32::new(Error::INTERNAL_START + 7).unwrap().into()
-}
-
-fn super_insecure_rng(buf: &mut [u8]) -> Result<(), Error> {
-    // `getrandom` guarantees it will not call any implementation if the output
-    // buffer is empty.
-    assert!(!buf.is_empty());
```

Comment:
I think it is useful to have a test that calls `getrandom::getrandom[_uninit()]` with an empty `dest`, and then also add this assertion to our insecure/mock custom implementation.

Reply:
Added, good point.